src/components/dialogs/error.vue

```diff
@@ -14,6 +14,15 @@
           For assistance, please contact BC Registries staff:
         </v-card-text>
       </div>
+      <div v-if="isExists">
+        <v-card-text class="copy-normal pt-8">
+          You Have Already created a request with same name.
+          Please go back and change the names , or in order to edit please use manage my request tab.
```

Comment:
The capitalization on line 19 looks incorrect.

There should be no space before the comma on line 20.

I think the tab name should be capitalized on line 20.

Did the UI/UX team provide any input for this? Maybe double-check with them.

Reply:
> The capitalization on line 19 looks incorrect.
> 
> There should be no space before the comma on line 20.
> 
> I think the tab name should be capitalized on line 20.
> 
> Did the UI/UX team provide any input for this? Maybe double-check with them.

There is another ticket also need to work on error messages. I talked to Yui and he suggested to open a separate ticket for the error messages.

Reply:
Isn't **this** a ticket for error messages?

Anyway, if you can't get an answer right away, please capitalize the tab name as per my previous comment.